extensions/amp-accordion/1.0/amp-accordion.js

```diff
@@ -34,9 +34,20 @@ const HEADER_SHIM_PROP = '__AMP_H_SHIM';
 const CONTENT_SHIM_PROP = '__AMP_C_SHIM';
 const SECTION_POST_RENDER = '__AMP_PR';
 
+/** @extends {PreactBaseElement<AccordionDef.AccordionApi>} */
 class AmpAccordion extends PreactBaseElement {
   /** @override */
   init() {
+    this.registerApiAction('toggle', (api, invocation) =>
+      api./*OK*/ toggle(invocation.args && invocation.args['section'])
+    );
+    this.registerApiAction('expand', (api, invocation) =>
+      api./*OK*/ expand(invocation.args && invocation.args['section'])
```

Comment:
Will you be adding firing AMP events in a separate PR?

Reply:
Thanks @caroqliu! It's an important topic. If a separate PR, let's make sure we track.

Reply:
Thank yes, this will be in a separate PR!